great_expectations/agent/actions/run_onboarding_data_assistant.py

```diff
@@ -0,0 +1,86 @@
+from typing import TYPE_CHECKING
+
+from great_expectations.agent.actions.action import (
+    ActionResult,
+    AgentAction,
+    CreatedResource,
+)
+from great_expectations.agent.models import RunOnboardingDataAssistantEvent
+from great_expectations.exceptions import StoreBackendError
+
+if TYPE_CHECKING:
+    from great_expectations.core import ExpectationSuite
+
+
+class RunOnboardingDataAssistantAction(AgentAction):
+    def run(self, event: RunOnboardingDataAssistantEvent, id: str) -> ActionResult:
+        expectation_suite_name = f"{event.data_asset_name} onboarding assistant suite"
```

Comment:
Probably doesn't matter but should we use `_` instead of spaces?

Reply:
since this is a value prominently displayed in the UI i think spaces are cleaner - happy to update if there is more concern around this, but as far as i know we fully support spaces in resource names.